drivers/fuel_gauge/max17048/Kconfig

```diff
@@ -14,7 +14,8 @@ config MAX17048
 
 config EMUL_MAX17048
 	bool "Emulate an MAX17048 fuel gague"
-	depends on EMUL
+	default y
+	depends on EMUL && DT_HAS_MAXIM_MAX17048_ENABLED
```

Comment:
Why is this `DT_HAS_MAXIM_MAX17048` and not `MAX17048`, which would match the pattern of the other emulators?

Reply:
Because I changed my mind to prefer the `MAX17048` pattern but I guess forgot to fix this one. Thank you for the catch!